src/scripts/main.js

```diff
@@ -1 +1,11 @@
 'use strict';
+
+const mainImage = document.getElementById('largeImg');
+const gallery = document.querySelector('.gallery__list');
+
+for (const item of gallery.querySelectorAll('.list-item')) {
+  item.addEventListener('click', () => {
+    event.preventDefault();
+    mainImage.src = item.children[0].href;
+  });
+}
```

Comment:
Проблема через глобальну змінну windows.event
Використай іншу назву, наприклад:

```suggestion
for (const item of gallery.querySelectorAll('.list-item')) {
  item.addEventListener('click', (e) => {
    e.preventDefault();
    mainImage.src = item.children[0].href;
  });
}
```

Reply:
Привіт, Назаре! Дякую, за рев'ю та допомогу! Зробив